File: grupo11_BuscaLinear/codigo/python/busca_linear.py
def buscaLinear(lista_para_buscar, valor_procurado):
  for i in range(len(lista_para_buscar)):
    
    elemento_atual = lista_para_buscar[i]
    
    if elemento_atual == valor_procurado:
      return i  
  
  return None

File: grupo11_BuscaLinear/codigo/java/BuscaLinear.java
// Implementação do algoritmo de Busca Linear.
public class BuscaLinear {
    public static int buscar(int[] vetor, int valorProcurado) {
        for (int i = 0; i < vetor.length; i++) {
            if (vetor[i] == valorProcurado) {
                return i;
            }
        }
        return -1;
    }
}
